app/src/main/java/com/nexters/moss/ui/receive/ReceiveActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.nexters.moss.ui.receive
+
+import android.app.AlertDialog
+import android.os.Bundle
+import android.view.ContextThemeWrapper
+import android.view.Gravity
+import android.widget.RadioGroup
+import android.widget.Toast
+import androidx.lifecycle.Observer
+import com.nexters.moss.R
+import com.nexters.moss._base.BaseActivity
+import com.nexters.moss.databinding.ActivityReceiveBinding
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class ReceiveActivity : BaseActivity<ActivityReceiveBinding>() {
+    override val vm: ReceiveViewModel by viewModel()
+    override fun getLayoutRes() = R.layout.activity_receive
+    override fun setupBinding() {
+        binding.vm = vm
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        vm.report.observe(this, Observer {
+            if (vm.report.value!!) {
+                showDialog()
+            }
+        })
+
+    }
+
+    private fun showDialog() {
+
+        val dialogView = layoutInflater.inflate(R.layout.layout_dialog_report, null)
+
+        dialogView.findViewById<RadioGroup>(R.id.radio_group_report)
+            .apply {
+                setOnCheckedChangeListener { _, id ->
+                    when (id) {
+                        R.id.btn_report_offensive -> toast("1")
```

Comment:
https://stackoverflow.com/questions/49194398/how-to-use-two-way-data-binding-with-radio-button-in-android

Reply:
@YChaeeun